funnel/views/space.py

```diff
@@ -141,3 +140,18 @@ def space_edit(profile, space):
         flash(_("Your changes have been saved"), 'info')
         return redirect(space.url_for(), code=303)
     return render_form(form=form, title=_("Edit proposal space"), submit=_("Save changes"))
+
+@app.route('/<space>/rsvp', methods=['POST'], subdomain='<profile>')
+@load_models(
+    (Profile, {'name': 'profile'}, 'g.profile'),
+    ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
+    permission='view')
+def rsvp(profile, space):
+    rsvp = RSVP.query.filter_by(proposal_space_id=space.id, user_id=g.user.id).first()
+    if rsvp:
+        rsvp.rsvp_action = request.form['rsvp_action']
+    else:
+        rsvp = RSVP(proposal_space_id=space.id, user_id=g.user.id, rsvp_action=request.form['rsvp_action'])
```

Comment:
When you make a new entity, pass it objects, not ids. Use `proposal_space=space, user=g.user, rsvp_action=form.rsvp.data`